test-adapter/src/main/scala/org/scalajs/testadapter/package.scala

```diff
@@ -12,8 +12,13 @@ package org.scalajs
 import scala.concurrent.{Await, Future}
 import scala.concurrent.duration._
 
+import org.scalajs.linker.ModuleInitializer
+
 package object testadapter {
   private[testadapter] implicit class AwaitFuture[T](val t: Future[T]) extends AnyVal {
     def await(): T = Await.result(t, Duration.Inf)
   }
+
+  def testAdapterInitializer: ModuleInitializer =
+    ModuleInitializer.mainMethod("org.scalajs.testinterface.Bridge", "start")
```

Comment:
I don't really like this package level stuff. WDYT? Should I make an object holder for it instead?

Reply:
Yeah, I was also surprised to see this here. An object holder would probably be better.

And actually, as the CI discovered, you cannot declare this `ModuleInitializer` here, because `testAdapter` does not know about the `linker`. You have to leave the construction of the `ModuleInitializer` to the sbt plugin, and expose the two strings independently.

So maybe the holder could look like:
```scala
object TestAdapterInitializer {
  val ModuleClassName: String = "org.scalajs.testinterface.Bridge"
  val MainMethodName: String = "start"
}
```